src/MeetingsSDKAdapter.js

```diff
@@ -1472,9 +1428,12 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
       throw new Error(`Could not find meeting with ID "${ID}"`);
     }
 
-    const updates = await updater(meeting);
-    const updatedMeeting = {...meeting, ...updates};
+    const updates = await updater(meeting, sdkMeeting);
+
+    deepMerge(meeting, updates);
+
+    this.meetings[ID] = meeting;
```

Comment:
no need for this, meeting is already referencing `this.meeting[ID]` hence it is updated when props are changed in `meeting`